client/src/components/answers/TopAnswer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+
+const TopAnswer = props => {
```

Comment:
I hear ya, but I think this is a git thing, or maybe you can help me make it look the same in git as in my editor, but I don't see that and have 2 spaces:
![image](https://user-images.githubusercontent.com/14282372/28982587-41cfbf94-790b-11e7-8ff6-46a791c71414.png)